### ISSUE TITLE
How to upgrade shared libraries?

### ISSUE THREAD

**Issue by pawamoy** [pipxproject/pipx#396]
Related to #365, I want to try installing a local package (or a remote one with a git spec, while in the directory of a local clone), however the pip version if the venv created by pipx is still v20.0.2, coming from the shared libraries.

```
pipx > (run_subprocess:112): running /home/pawamoy/.local/pipx/.cache/ea291b3e2cdfc98/bin/python -m pip --version
pip 20.0.2 from /home/pawamoy/.local/pipx/shared/lib/python3.8/site-packages/pip (python 3.8)
```

I updated my system Python pip with `sudo pip install -U pip` (`Successfully installed pip-20.1`), but the shared pip is still 20.0.2. Is there a way to upgrade the pipx shared libraries?

**Comment by uranusjr** [comment]
See #318

**Comment by pawamoy** [comment]
Thanks!

TL;DR: `~/.local/pipx/shared/bin/pip install -U pip` and this is done automatically every month thanks to #318.
